net-voip/sipvicious/sipvicious-0.3.4.ebuild

```diff
@@ -0,0 +1,30 @@
+# Copyright 2020-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{7..10} )
+DISTUTILS_USE_SETUPTOOLS=rdepend
+DISTUTILS_SINGLE_IMPL=1
+
+inherit distutils-r1
+
+DESCRIPTION="SIPVicious is a SIP security package"
+HOMEPAGE="https://github.com/EnableSecurity/sipvicious/wiki"
+SRC_URI="https://github.com/EnableSecurity/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+KEYWORDS="~amd64 ~x86"
+
+LICENSE="GPL-3+"
+SLOT="0"
+
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
```

Comment:
With distutils-r1 you don't need to define this, only needed with python-* eclasses (or if you set DISTUTILS_OPTIONAL, but it's not here)